README.md

```diff
@@ -95,7 +95,11 @@ This macro implements an "outer union." The list of tables provided to this macr
 
 Usage:
 ```
-{{ macro union_tables(tables=[ref('table_1', 'table_2')], column_override={"some_field": "varchar(100)"}) }}
+{{ macro union_tables(
```

Comment:
can you make this show
```
{{ dbt_utils.union_tables(
    tables=[ref('table_1'), ref('table_2')], 
    column_override={"some_field": "varchar(100)"}, 
    exclude=["some_other_field"]
) }}
```

^ that will fix a typo, plus, macros now need to be scoped with the name of the package